### PR TITLE
use _names suffix when only returning organisms' name

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

https://github.com/AlexsLemonade/refinebio-frontend/pull/825

## Purpose/Implementation Notes

We tend to interchangeably use `organisms` and `organism_names`

This PR intends on making the use of `organism_name` or `organism_names` when the attribute does not contain a serialized model but instead contains only a string of the name. ie `HOMO_SAPIENS`

## Types of changes

What types of changes does your code introduce?

- Bugfix (non-breaking change which fixes an issue)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Functional tests

Loading experiments locally

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: api/data_refinery_api/serializers.py [hunk 1]
```diff
@@ -447,7 +447,7 @@ class DetailedExperimentSerializer(serializers.ModelSerializer):
     annotations = ExperimentAnnotationSerializer(many=True, source="experimentannotation_set")
     samples = DetailedExperimentSampleSerializer(many=True)
     sample_metadata = serializers.ReadOnlyField(source="sample_metadata_fields")
-    organisms = serializers.StringRelatedField(many=True, read_only=True)
+    organism_names = serializers.StringRelatedField(many=True, source="organisms", read_only=True)
```
Comment (kurtwheeler) [inline api/data_refinery_api/serializers.py]: I would expect this endpoint to return the full Organism model instead of just the name.

Comment (davidsmejia) [reply api/data_refinery_api/serializers.py]: This would mean we would need to change the search document serializer to also return the object as well.

Comment (davidsmejia) [reply api/data_refinery_api/serializers.py]: https://github.com/AlexsLemonade/refinebio/issues/2055

**Comment by davidsmejia** [comment]
Oh right on, will fix before merging!

**Pull request merged**